bruno/collections/Rafiki/Examples/Open Payments Without Quote/Create Outgoing Payment.bru

```diff
@@ -0,0 +1,49 @@
+meta {
+  name: Create Outgoing Payment
+  type: http
+  seq: 9
+}
+
+post {
+  url: {{senderOpenPaymentsHost}}/outgoing-payments
+  body: json
+  auth: none
+}
+
+headers {
+  Authorization: GNAP {{accessToken}}
+}
+
+body:json {
+  {
+      "walletAddress": "{{senderWalletAddress}}",
+      "incomingPayment": "{{incomingPaymentUrl}}",
+      "debitAmount": {{quoteDebitAmount}},
```

Comment:
should we just hardcode this, to remove any references to quote?

Reply:
It's used in a couple places (grant request and create outgoing payment) so i'd prefer not to hardcode but point taken on the name. I changed to `debitAmount`.